lib/reference/functions.js

```diff
@@ -16,7 +16,22 @@ const camelCaseFunctions = new Set([
 
 const mathFunctions = new Set(['calc', 'clamp', 'max', 'min']);
 
+const colorFunctions = new Set([
+	'rgb',
+	'rgba',
+	'hsl',
+	'hsla',
+	'hwb',
+	'lab',
+	'lch',
+	'oklab',
+	'oklch',
+	'color',
+	'color-mix',
+]);
```

Comment:
It seems like it's missing `gray`.

cf
https://github.com/stylelint/stylelint/blob/40b4a6daa6168201e55eedae2acef431e216978b/lib/rules/color-named/README.md?plain=1#L22
https://github.com/stylelint/stylelint/blob/40b4a6daa6168201e55eedae2acef431e216978b/lib/rules/color-named/colordUtils.js#L22

Reply:
`gray` is not a color function :)

I think it was proposed at some point, but it never shipped in any browser.
https://github.com/postcss/postcss-color-gray